### PR TITLE
Attempting to define a new filehandler for HDF5plugin Single mode

### PR DESCRIPTION
**Pull request by awalter-bnl**
This should add a new file handler that will hopefully allow for the XIP data to be retrieved by databroker.
It also includes several changes to meet pep8 standards

Specific questions I have: 
is line 26 correct?
is line 42 correct (I think that filename may actually include the path but am not sure)?

@danielballan or @tacaswell your feedback would be appreciated
@stuwilkins you mentioned that you wrote the previous hdf5 handlers, having you look over this would also be useful.

### CODE REVIEW

**Comment by awalter-bnl** [comment]
@stuwilkins I think I have resolved all of the comments, let me know.

File: startup/22-rixscam.py [hunk 1]
```diff
@@ -16,18 +16,25 @@
...
+    key : string
+        the 'path' inside the file to the data set.
+    frame_per_point : float
+        the number of frames per point.
     '''
 
-    specs = {AD_HDF5_SINGLE} | HandlerBase.specs
+    specs = {'HDF5_SINGLE'} | HandlerBase.specs
 
     def __init__(self, fpath, template, filename, frame_per_point=1):
```
Comment (stuwilkins) [inline startup/22-rixscam.py]: I think this bit is still wrong.

Comment (stuwilkins) [reply startup/22-rixscam.py]: It should be passed as a kwarg

Comment (awalter-bnl) [reply startup/22-rixscam.py]: I have removed this from this class and left it in the other class, as I think this makes more sense. If we are going to seperate AD only parts then the name including AD should be added to this list.

**Pull request merged**